### PR TITLE
Bump the version of knative/pkg to something much more recent.

### PR DESCRIPTION
**Pull request by dlorenc**
# Changes

We're a few months behind and are missing quite a few bug fixes and performance improvements.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [X] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [X] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

### CODE REVIEW

File: pkg/logging/config.go [hunk 1]
```diff
@@ -51,16 +51,16 @@ func NewLoggerFromConfig(config *logging.Config, name string) (*zap.SugaredLogge
...
 // NewConfigFromMap creates a LoggingConfig from the supplied map
 func NewConfigFromMap(data map[string]string) (*logging.Config, error) {
-	return logging.NewConfigFromMap(data, components...)
+	return logging.NewConfigFromMap(data)
 }
 
 // NewConfigFromConfigMap creates a LoggingConfig from the supplied ConfigMap
 func NewConfigFromConfigMap(configMap *corev1.ConfigMap) (*logging.Config, error) {
-	return logging.NewConfigFromConfigMap(configMap, components...)
+	return logging.NewConfigFromConfigMap(configMap)
```
Comment (ImJasonH) [inline pkg/logging/config.go]: These methods can just go away now right?

Comment (dlorenc) [reply pkg/logging/config.go]: Yup, cleaned up now.

**Pull request merged**